systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaTopicResource.java

```diff
@@ -77,7 +77,7 @@ private static KafkaTopic getKafkaTopicFromYaml(String yamlPath) {
     }
 
     private static KafkaTopic waitFor(KafkaTopic kafkaTopic) {
-        return ResourceManager.waitForResourceStatus(kafkaTopicClient(), kafkaTopic, "Ready");
+        return ResourceManager.waitForResourceStatus(kafkaTopicClient(), kafkaTopic, "Ready", 1);
```

Comment:
This is a little bit confusing the fourth argument is expectPods, and you are excepting to have `KafkaTopic` one pod? Please describe and same with the user.